internal/cluster/transition.go

```diff
@@ -562,3 +572,20 @@ func setPendingUserReset(ctx context.Context, c *common.Cluster, db *gorm.DB, ho
 	txSuccess = true
 	return nil
 }
+
+func addExtraParams(log logrus.FieldLogger, cluster *common.Cluster, clusterState string) ([]interface{}, error) {
+	extra := []interface{}{}
+	switch clusterState {
+	case statusInfoInstalling:
```

Comment:
I think here the case value should be `models.HostStatusInstalling` if using `clusterState` in switch